node/integration_test.go

```diff
@@ -126,6 +126,84 @@ func TestTxGossipingAndAggregation(t *testing.T) {
 
 	for i, app := range apps {
 		app.AssertNumberOfCalls(t, "DeliverTx", clientNodes)
+		app.AssertNumberOfCalls(t, "GenerateFraudProof", 0)
+
```

Comment:
Removed `app.AssertExpectations(t)` here because `GenerateFraudProof` is not called. Is that okay?